scene/gui/tab_container.cpp

```diff
@@ -371,8 +371,10 @@ void TabContainer::_notification(int p_what) {
 			// Draw the tab area.
 			panel->draw(canvas, Rect2(0, header_height, size.width, size.height - header_height));
 
-			// Draw selected tab in front
-			_draw_tab(tab_fg, font_color_fg, current, tabs_ofs_cache + x_current);
+			// Draw selected tab in front. Need to check tabs.size() in case of no contents at all.
```

Comment:
Can you simply check if `tabs.size() > 0` then ? This check is quite weird, because outside of the case where the size is 0, this condition is always true.

Reply:
Shouldn't we abort way earlier if `tabs.size()` is 0? Is there actually anything to draw beyond https://github.com/godotengine/godot/pull/43173/files#diff-b799a8e076432728cb4a5a81555f980698cc5dd9ac48c62dcf40303473b5c2a3R263-R268 ?

Reply:
Yes, that would likely be cleaner, I agree.

Reply:
tab area should be drawn at previous line to correct order of tabs and content area.
```
line 372 : panel->draw(canvas, Rect2(0, header_height, size.width, size.height - header_height));
```
so I think `tab.size() > 0` would be best option.